config/pfblockerng/pfblockerng_diag_dns.php

```diff
@@ -73,7 +64,7 @@
 	$type = "hostname";
 	if ($resolved) {
 		$resolved = array();
-		exec("{$cmd} {$host_esc} A | /usr/bin/grep {$host_esc} | /usr/bin/grep -v ';' | /usr/bin/awk '{ print $5 }'", $resolved);
+		exec("/usr/bin/drill {$host_esc} A | /usr/bin/grep {$host_esc} | /usr/bin/grep -v ';' | /usr/bin/awk '{ print $5 }'", $resolved);
```

Comment:
It would be nice to replace exec() by a native PHP function, http://php.net/manual/en/function.dns-get-record.php